lib/swoosh/attachment.ex

```diff
@@ -3,7 +3,7 @@ defmodule Swoosh.Attachment do
   Struct representing an attachment in an email.
   """
 
-  defstruct filename: nil, content_type: nil, path: nil
+  defstruct filename: nil, content_type: nil, path: nil, type: nil
```

Comment:
- Maybe just `inline: false` (default to attachment)?
- Should we do something special about inline attachments? e.g. validating it's an image?

Reply:
I'm ok with using `type`. Slightly more future-proof.

@princemaple: regarding validation of inline attachments, do you know if the spec says anything about accepted file type for inline attachments?

Reply:
I actually failed to search for the inline attachment related specs. :(